scripts/integration-test

```diff
@@ -14,4 +14,4 @@ if [ ! -e ../dist/artifacts/initrd ]; then
     ../scripts/dev
 fi
 
-go test -timeout 30m
+go test -v -check.vv -timeout 4h
```

Comment:
But when I try to create new PR - I am getting to view this one, no option to create new one. Sorry I am new in github tricks.